timesketch/frontend/dist/login.html

```diff
@@ -15,4 +15,8 @@
         .card-content {
             justify-content: center;
             align-items: center;
-        }</style><link href=/dist/css/chunk-common.73b6326f.css rel=preload as=style><link href=/dist/css/chunk-vendors.52c8c6fd.css rel=preload as=style><link href=/dist/js/chunk-common.90aeddb5.js rel=preload as=script><link href=/dist/js/chunk-vendors.0cb9db01.js rel=preload as=script><link href=/dist/js/login.c1fa2383.js rel=preload as=script><link href=/dist/css/chunk-vendors.52c8c6fd.css rel=stylesheet><link href=/dist/css/chunk-common.73b6326f.css rel=stylesheet></head><body class=login-page><div class=columns><div class=column><div class="card is-wide has-text-centered"><div class=card-content><div class=content><div><img src=/dist/timesketch-color.png style=width:40px;><div style="color: #333; font-size: 1.5em;">time<b>sketch</b></div><div style="color: #666; font-size: 0.8em;">Digital Forensic Timeline Analysis</div></div><br><form method=post style="width:50%;margin-left: auto; margin-right: auto;"><div class=field><div class=control><input type=text class=input name=username placeholder=Username style=text-align:center;></div></div><div class=field><div class=control><input type=password class=input name=password placeholder=Password style=text-align:center;></div></div><div class=field><div class="control has-text-centered"><button type=submit class="button is-rounded is-wide is-info" style=width:100%;>Sign in</button></div></div>{{ form.csrf_token }}</form></div></div></div></div></div><script src=/dist/js/chunk-vendors.0cb9db01.js></script><script src=/dist/js/chunk-common.90aeddb5.js></script><script src=/dist/js/login.c1fa2383.js></script></body></html>
\ No newline at end of file
+<<<<<<< HEAD
```

Comment:
I'm assuming the <<< JHEAD is still an artifact from merging...

Reply:
fixed